MAINTAINERS.md

```diff
@@ -8,8 +8,8 @@ maintainer responsibilities.
 Maintainers are listed in alphabetical order.
 
 * Jose Cortes, [josecv](https://github.com/josecv), <josecortes@datawire.io>
+* Kévin Lambert, [knlambert](https://github.com/knlambert), <klambert@datawire.io>
```

Comment:
Adding Kévin now is a bit too early if we're following our governance rules (mentioned higher up in this doc).